runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java

```diff
@@ -30,355 +30,10 @@
 import java.util.Set;
 import lombok.Getter;
 import net.runelite.api.EquipmentInventorySlot;
-import static net.runelite.api.EquipmentInventorySlot.AMULET;
-import static net.runelite.api.EquipmentInventorySlot.BODY;
-import static net.runelite.api.EquipmentInventorySlot.BOOTS;
-import static net.runelite.api.EquipmentInventorySlot.CAPE;
-import static net.runelite.api.EquipmentInventorySlot.GLOVES;
-import static net.runelite.api.EquipmentInventorySlot.HEAD;
+import static net.runelite.api.EquipmentInventorySlot.*;
```

Comment:
Is every slot being imported before this commit? It doesn't look like it.

Anyway, it looks like you missed removing the `LEGS` import. :question: